src/components/map/overview-table.tsx

```diff
@@ -294,29 +299,44 @@ export function OverviewTable({
           <TableHeader>
             {table.getHeaderGroups().map((headerGroup) => (
               <TableRow key={headerGroup.id}>
-                {headerGroup.headers.map((header) => {
-                  return (
-                    <TableHead key={header.id}>
-                      <TooltipProvider>
-                        <Tooltip>
-                          <TooltipTrigger>
-                            {header.isPlaceholder
-                              ? null
-                              : flexRender(
-                                  header.column.columnDef.header,
-                                  header.getContext()
-                                )}
-                          </TooltipTrigger>
-                          <TooltipContent>
-                            {tooltips[
-                              header.column.id as keyof typeof tooltips
-                            ] || ""}
-                          </TooltipContent>
-                        </Tooltip>
-                      </TooltipProvider>
-                    </TableHead>
-                  );
-                })}
+                {headerGroup.headers.map((header) => (
+                  <TableHead key={header.id}>
+                    <TooltipProvider>
+                      <Tooltip>
+                        <TooltipTrigger>
+                          {header.isPlaceholder ? null : header.column.getCanSort() ? (
+                            <Button
+                              variant="ghost"
+                              onClick={header.column.getToggleSortingHandler()}
+                              className="h-max p-1 w-min w-full"
+                            >
+                              {flexRender(
+                                header.column.columnDef.header,
+                                header.getContext()
+                              )}
+                              {{
+                                asc: <ChevronUpIcon className="min-w-5 w-5" />,
+                                desc: (
+                                  <ChevronDownIcon className="min-w-5 w-5" />
+                                ),
+                              }[header.column.getIsSorted() as string] ?? null}
```

Comment:
We should include a default state to avoid layout shift when sorting the table. It would be better UX to make sure the columns do not change width when sorting. You could even include empty padding instead of an icon, play around with it and see what works best.